### PR TITLE
Image controller added

### PR DESCRIPTION
**Pull request by tpardeshi**

### CODE REVIEW

File: app/services/image_service.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module ImageService
+  class << self
```
Comment (rahulbajaj0509) [inline app/services/image_service.rb]: Why is this a class method?

Comment (tpardeshi) [reply app/services/image_service.rb]: While doing it, I came across two ways of writing services. 
1. Service Objects as modules
2. Service Objects as classes
From which I tried [1] approach.

**Comment by tpardeshi** [comment]
@rahulbajaj0509 In recent commit I am getting a  error for file upload "NoMethodError (undefined method `original_filename' for nil:NilClass)". I am still trying to solve it out. If in case you are familiar with the error please do suggest.

**Comment by tpardeshi** [comment]
Issue in the comment@https://github.com/lingostack/lingoqa/pull/5#issuecomment-555985468 is been solevd and pushed.

**Comment by rahulbajaj0509** [comment]
Can you solve the merge conflicts :+1:

**Comment by tpardeshi** [comment]
@rahulbajaj0509  From my side I see no conflicts. It says "This branch has no conflicts with the base branch".

**Pull request closed without merging**